lib/zlib.js

```diff
@@ -800,8 +801,8 @@ function Brotli(opts, mode) {
   assert(mode === BROTLI_DECODE || mode === BROTLI_ENCODE);
 
   TypedArrayPrototypeFill(brotliInitParamsArray, -1);
-  if (opts && opts.params) {
-    for (const origKey of ObjectKeys(opts.params)) {
+  if (opts?.params) {
+    ArrayPrototypeForEach(ObjectKeys(opts.params), (origKey) => {
```

Comment:
What are your thoughts on using `for(;;)` over switching it to `ArrayPrototypeForEach`? From the performance point of view...

Reply:
We would need to benchmark that, my guess is that both approaches are equivalent.

Reply:
According to our benchmark, this change doesn't impact Node.js performance. If you want, you can open a PR proposing to use a `for(;;)` loop instead, but tbh I'd be surprised if it mattered at all.